### DISCUSSION TITLE
CoreML models

### DISCUSSION THREAD

**Discussion by john-odysseyapp** [OpenModelDB/open-model-database#311] [General]
First off this has been far and away the best resource I've found about upscalers so thank you so much for pulling this together. I was wondering if there was any way to filter or see which models have been converted to CoreML? We're looking for a few models to add to a Mac app and I'm having trouble finding ones that are state of the art and set up to work on a Mac. Thank you!!

**Comment by joeyballentine** [comment]
Hi, thanks for the kind words. None of the models we host have been converted to CoreML. As far as I'm aware, CoreML is rather locked down and it isn't easy to convert models to it. Even the CoreML onnx runtime doesn't support certain operations that would be needed to run these models.

If you have any resources that would indicate otherwise though, please do share, as I'm not very familiar with CoreML generally.

**Comment by john-odysseyapp** [comment]
Thanks for getting back to me. That’s a bummer about no converted models. I believe it’s gotten a bit easier (there are some details on HuggingFace here <https://huggingface.co/coreml-community>) but it’s a bit time consuming. We are building a Mac app called Odyssey: https://odysseyapp.io/ and we’re trying to add in a whole bunch of upscalers but we’re hitting a bit of a wall.

Any way thanks for hosting a great community - it has been really useful as we’re trying to figure out what models we want to try and support.

Thanks
John
